_posts/0000-01-02-nafis19.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text is not correct!!!